static/monorail-2.0.yaml

```diff
@@ -4680,6 +4680,39 @@ paths:
       - Write
       - workflowsModify
     x-swagger-router-controller: workflows
+  /notification/alerts:
```

Comment:
@yyscamper from what I understand, the monorail-2.0-sb.yaml is not being used. I won't be adding the API there. It is actually going away.....https://rackhd.atlassian.net/browse/RAC-1447